packages/fxa-settings/src/pages/InlineRecoverySetup/index.tsx

```diff
@@ -159,6 +159,7 @@ const InlineRecoverySetup = ({
                   inputFtlId: 'inline-recovery-backup-authentication-code',
                   submitButtonText: 'Confirm',
                   submitButtonFtlId: 'inline-recovery-confirm-button',
+                  inputMode: InputModeEnum.text,
                   pattern: '\\w{10}',
```

Comment:
Noticed the pattern used [here](https://github.com/mozilla/fxa/blob/8364d55f01ca6bb41515aba490558e39cc0e6399/packages/fxa-settings/src/pages/ResetPassword/ConfirmTotpResetPassword/index.tsx#L44) is slightly different. Seems like the latter might be better since `\w` includes underscore.

Reply:
@vbudhram Thanks, I used that and made a shared constant between them.